src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -274,8 +276,10 @@ thus, rotating the model in WebGL scene.
   <figcaption><b>Figure 4</b>: Sensor updates orientation of a 3D model</figcaption>
 </figure>
 
-The following example calculates the maximum velocity of a device from a linear
-acceleration sensor assuming that the device was initially still.
+The following extract from [punchmeter demo]
```

Comment:
The following code snippet is extracted from the ... , illustrating how .. under the assumption that it is initially laying still